### PR TITLE
Update cancel-your-subscription.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line stating that you can reduce it to less than 25 and proceed with cancellation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3727

File: microsoft-365/commerce/subscriptions/cancel-your-subscription.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ ms.date:
 
 ::: moniker-end
 
-*Eligibility:* If you have fewer than 25 licenses assigned to users, you can cancel your Microsoft 365 for business trial or paid subscription online in the Microsoft 365 admin center at any time. If you have more than 25 licenses assigned to users, [call support to cancel your subscription](../../admin/contact-support-for-business-products.md).
+*Eligibility:* If you have fewer than 25 licenses assigned to users, you can cancel your Microsoft 365 for business trial or paid subscription online in the Microsoft 365 admin center at any time. If you have more than 25 licenses assigned to users, reduce it to less than 25 or  [call support to cancel your subscription](../../admin/contact-support-for-business-products.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/cancel-your-subscription.md]:
```suggestion
*Eligibility:* If you have fewer than 25 licenses assigned to users, you can cancel your Microsoft 365 for business trial or paid subscription online in the Microsoft 365 admin center at any time. If you have more than 25 licenses assigned to users, reduce it to less than 25 or [call support to cancel your subscription](../../admin/contact-support-for-business-products.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by cmcatee-MSFT** [comment]
Wait on response from the product team.

**Pull request merged**